gen/definitions/interface_ethernet.yaml

```diff
@@ -430,6 +430,14 @@ attributes:
       - yang_name: direction
         id: true
         example: input
+  - yang_name: load-interval
+    example: 30
+  - yang_name: Cisco-IOS-XE-snmp:snmp/trap/link-status
+    write_only: true
```

Comment:
Why is this `write_only`? Is it not included in the GET response?

Reply:
Originally I thought it was not worth the troubleshooting effort (a good few hours), but since you've raised a question I did spend some time on it. It seems that the culprit is `source_template` which overrides some (all?) settings.

Hence I propose to exclude it from tests, to avoid further confusion. I'd pushed a commit to that effect. Sounds good?